### PR TITLE
Gatsby V3

### PR DESCRIPTION
**Pull request by tnorlund**
I'd really like to get this working with Gatsby V3. I'm having issues with the Jest suite. I'm not sure how to generate another snapshot in` __snapshots__`. Could you help me out?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,13 +50,14 @@
     "mdast-util-to-hast": "^6.0.2",
     "open": "^7.0.0",
     "prettier": "^1.18.2",
+    "react": "^17.0.1",
```
Comment (andrewbranch) [inline package.json:53]: Accidental addition?

**Review by andrewbranch — COMMENTED** [review]
Thanks for the help! Running `yarn test -u` should update all the baselines. There are some [instructions in CONTRIBUTING.md](https://github.com/andrewbranch/gatsby-remark-vscode/blob/master/CONTRIBUTING.md#tests).

**Comment by tnorlund** [comment]
All of the GraphQL resolvers are failing. Does the `yarn test -u` change the expectation of the tests?

**Comment by andrewbranch** [comment]
My bad, react is now a peerDependency of gatsby, so it does need to be installed. The issue is that gatsby's local server (used to test GraphQL queries) isn't spinning up at all. I'm taking a look and will push an update.

**Comment by andrewbranch** [comment]
@tnorlund, can you check the box on the PR that allows maintainers to make edits? That will let me push a commit.

**Comment by tnorlund** [comment]
> @tnorlund, can you check the box on the PR that allows maintainers to make edits? That will let me push a commit.

It's already checked. Do you want me to uncheck it?

**Comment by andrewbranch** [comment]
Oh, I think it's because it's on the master branch of your fork. I think the easiest thing to do would be to give me push access to your fork. Otherwise I'll have to PR into your PR.

**Comment by tnorlund** [comment]
I added you as a collaborator. Does that work?

**Comment by andrewbranch** [comment]
That worked, thanks!

**Review by andrewbranch — COMMENTED** [review]
Ok, tests are working, which is a good sign. I think before merging, I would like to get the example site updated to v3 as well just so we can see that it works fully integrated in a real gatsby environment. Would you mind looking into that? I really don’t care whether it’s upgraded in place or recreated with `gatsby new` and the content and config dropped back in. Whatever you think is easier, if you’re up for it.

**Comment by tnorlund** [comment]
Are you referring to the site in `examples/example-site/`?

**Comment by andrewbranch** [comment]
Yep, that’s the one.

**Comment by tnorlund** [comment]
Alright. I'm going to make the changes required for Gatsby V3.

**Comment by tnorlund** [comment]
Tests pass locally, fail in GH Action. Any idea why?

**Comment by tnorlund** [comment]
If I can help with anything else, please let me know!

**Comment by tnorlund** [comment]
I guess that's it. I'd like to add this to my blog once it's been merged. It's a neat plugin.

**Comment by andrewbranch** [comment]
Thanks so much!

**Pull request merged**